client/src/utils/COLORS.js

```diff
@@ -0,0 +1,7 @@
+const colors = {
+  primaryYellow: '#F8F00E',
```

Comment:
You only have primary colors :). 
Instead of naming the property like the color (`Yellow`), it's better to find a more generic name.